### ISSUE TITLE
vips script return  vip: line 93: args: readonly variable

### ISSUE THREAD

**Issue by ardoli** [molgenis/vip#321]
**Describe the bug**
Script vip.sh return the error : vip: line 93: args: readonly variable

**To Reproduce**
Go to 
vip/test/resources
run 
../../vip -w vcf -i gvcf.tsv -o ~/testDir

**Desktop (please complete the following information):**
 - OS: AlmaLinux" VERSION="8.7 (Stone Smilodon)"
 - Version 5.0.4

**Solution I use**
Edit line 112 vip.sh
`local -r args=$(getopt -a -n pipeline -o w:i:o:c:p:rh --long workflow:,input:,output:,config:,profile:,resume,help -- "$@")
`
remove the "-r" for let variable args to be writable.
--> `local args=$(getopt -a -n pipeline -o w:i:o:c:p:rh --long workflow:,input:,output:,config:,profile:,resume,help -- "$@")`

it works for me

**Comment by dennishendriksen** [comment]
Hi @ardoli, thank you for reporting this issue.

I've managed to reproduce your issue with AlmaLinux 8.7. It boils down to:
```bash
#!/bin/bash
fn() {
  local args=()
}

main() {
  local -r args=()
  fn
}

main
```
resulting in:
```
[~]$ bash --version;bash test_readonly_args.sh
GNU bash, version 4.4.20(1)-release (x86_64-redhat-linux-gnu)
<cut>
test_readonly_args.sh: line 3: args: readonly variable
```
The same script runs successfully on a bunch of other Linux versions that I've tried. I'll come back to you after further investigation.

**Comment by dennishendriksen** [comment]
I've posted a question on the [Unix & Linux Stack Exchange](https://unix.stackexchange.com/questions/736448/bash-4-4-local-readonly-array-variable-scoping-bug).

**Comment by dennishendriksen** [comment]
@ardoli fixed in v5.0.5, please let me know if this doesn't resolve your issue

**Comment by ardoli** [comment]
Yes it's ok, the workaround is exactly the change I made in vip.sh script and it work.
thank's
